docs/resources/apig_api.md

```diff
@@ -69,9 +71,9 @@ The following arguments are supported:
 
 * `request_path` - (Required, String) Specifies the request address, which can contain a maximum of 512 characters
   request parameters enclosed with brackets ({}).
-  + The address can contain special characters, such as asterisks (), percent signs (%), hyphens (-), and
+  * The address can contain special characters, such as asterisks (), percent signs (%), hyphens (-), and
```

Comment:
[MD004 - Unordered list style](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md004---unordered-list-style) requires that each sublist has a consistent symbol that differs from its parent list.

Reply:
I agree on the understanding of the MD004 error but in my IDE or on the [demo site](https://dlaa.me/markdownlint/) the use of a `+` generates an error. See screenshot. 

![Capture d’écran 2022-12-15 à 09 11 24](https://user-images.githubusercontent.com/13660462/207808354-a91b02ed-f9e0-494b-8094-ed27c827a708.png)

![Capture d’écran 2022-12-15 à 09 20 31](https://user-images.githubusercontent.com/13660462/207808607-60b5caed-1d3d-496b-8e2f-ae4f413402d2.png)

Reply:
you should config the style of MD004 to "sublist"

```json
{
    "MD004": {
        "style": "sublist"
    }
}
```